backend/tests/views/annotation_campaign.py

```diff
@@ -85,6 +85,7 @@ def tearDown(self):
     def test_list(self):
         """AnnotationCampaign view 'list' returns list of campaigns"""
         url = reverse("annotation-campaign-list")
+        self.client.login(username="admin", password="osmose29")
```

Comment:
You should also test when the person is logged in and not staff :wink:. (also time to rebase)

Reply:
New tests are OK for me!